benchmark/data_frame_text_benchmark.py

```diff
@@ -73,6 +75,9 @@
     default="sentence-transformers/all-distilroberta-v1",
     choices=[
         "distilbert-base-uncased",
+        "roberta-large",
+        "microsoft/deberta-v3-large",
```

Comment:
For now maybe we can just follow the paper https://arxiv.org/pdf/2111.02705.pdf just include roberta and electra

Reply:
I've added DeBERTa, but you're right, I don't think we'll be running experiments with it unless necessary or unless we have plently of extra time, which usually doesn't happen.